samples/SimpleDataSource/SimpleCustomDataSource.cs

```diff
@@ -41,6 +41,65 @@ public class SimpleCustomDataSource
     {
         private IApplicationEnvironment applicationEnvironment;
 
+        //
+        // Provides information about this Custom Data Source, such as the author,
```

Comment:
>Custom Data Source [](http://example.com/codeflow?start=43&length=18)

Do we want to move away from Custom Data Source?

Reply:
https://github.com/microsoft/microsoft-performance-toolkit-sdk/issues/87